src/features/profile/Profile.module.scss

```diff
@@ -0,0 +1,23 @@
+.narratives {
```

Comment:
I think in order to complete the PR the CSS should be completed to an approximation of the current css. At least the dashed lines around the elements should be removed.

Reply:
We talked about this and made a decision as a team: [URO-99](https://kbase-jira.atlassian.net/browse/URO-99)